main.py

```diff
@@ -79,10 +79,10 @@ def add_handlers(dp: Dispatcher):
     dp.add_handler(CallbackQueryHandler(meme_callback, pattern=r"^meme_\.*"))
     dp.add_handler(CallbackQueryHandler(stats_callback, pattern=r"^stats_\.*"))
 
-    if config_map['meme']['comments']:
+    if Config.meme_get('comments'):
```

Comment:
I think this approach is reducing the potential of the config map. In this case the 'meme' category is hardcoded and you must define a new set of function for any new sub-category of configurations. This may be ok if we don't plan to modify the categories too much but consistently reduces the versatility.

Reply:
The Config class has a **get** method that allows for all the flexibility of a dictionary, the **meme_get** is just a commodity tool: in fact, it calls the **get** method under the hood.

Reply:
So is it necessary to have these commodity methods? I don't think they improve the readability too much nor are that shorter than the generic method.

Reply:
```py
# This is shorter
Config.meme_get('comments')
# than this
Config.settings_get('meme', 'comments')
```
Both methods are exactly the same and are currently available to the user, but since the "meme" settings are used very often, i though it could have been useful to provide a shorter, more specific method.